content/source/docs/enterprise/upgrade/differences.html.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: enterprise2
+page_title: "Current vs. Legacy - Upgrading - Terraform Enterprise"
+sidebar_current: "docs-enterprise2-upgrading-differences"
+---
+
+# Differences Between Current and Legacy Terraform Enterprise
+
+Terraform Enterprise began as part of the discontinued Atlas suite, and today's version is a ground-up redesign. If you've previously used the legacy version of Terraform Enterprise, we hope you enjoy the new version's improvements.
+
+Several things are significantly different in the new Terraform Enterprise. This page is a brief summary of what to expect when upgrading from the legacy version.
+
+## Renamed or Moved Features
+
+### Workspaces Replace Environments
+
+Environments are now called [workspaces](../workspaces/index.html). Our goal in this change was to make it easier to on-board new Terraform users in your organization by avoiding an overloaded and ambiguous term.
```

Comment:
I wonder if we could be stronger and just say "Using a dedicated term makes it easier to onboard..."

Reply:
Instead of the second sentence, maybe "Our goal in this change was to make the terminology used by Terraform Enterprise consistent with Terraform Open Source."

Reply:
I would also add something like: "Beyond the name change, the functionality of workspaces should be the same as what was previously known as 'environments.'"